release/content.sh

```diff
@@ -17,6 +17,15 @@ SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
 # ==============================================================================
 stage "Finalizing Content"
 
+if [[ -n ${LOCAL_RPM_PATH} && -d ${LOCAL_RPM_PATH} ]]; then
+    info "Copying local packages from ${LOCAL_RPM_PATH}"
+    cp "${LOCAL_RPM_PATH}"/*.rpm "${PKGS_DIR}"/
+    for file in "${LOCAL_RPM_PATH}"/*.rpm; do
```

Comment:
Instead of a loop, why not to just `cp <path>/*.rpm <dest>/`?

If you just want to list the rpms on the console output for tracking (which is nice if is not 800 or so), you can echo the package list file that needs to be created to make content.sh. 

content.sh is expecting only two things of content provides:
 * a txt file inside ${WORK_DIR} following this name pattern ${PKG_LIST_FILE}_<content provider name> (no extension)
 * RPMs to be placed inside ${PKGS_DIR}

Conflicts, overrides etc are not handled atm. It is the user's job to ensure that a valid set of RPMs is inside ${PKGS_DIR} folder.

Anyhow, there lots of room for improvements in this segment of code. This is definitely a step in this direction.